js-exercises/update-object/updateObject.js

```diff
@@ -0,0 +1,15 @@
+function updateObject(index, valueToReplace, arr) {
+  const arrayToUpdate = arr;
```

Comment:
Why not put this name (`arrayToUpdate`) in the parameters itself?

Reply:
Also, this thing does not make a copy of original array. Make a copy, change that, and return.

Reply:
At first i didn't have  `arrayToUpdate` but i was getting error `Assignment to property of function parameter 'arr'  no-param-reassign` that's why i have to reassign to commit, because of error i was not able to commit the code